eclipse-collections-testutils/src/test/java/org/eclipse/collections/impl/test/AssertNotEqualsThrowsTest.java

```diff
@@ -38,7 +42,7 @@ public void objectObject()
         }
         catch (AssertionError e)
         {
-            Verify.assertContains(AssertNotEqualsThrowsTest.class.getName(), e.getStackTrace()[0].toString());
+            Verify.assertContains("org.junit.Assert.fail", e.getStackTrace()[0].toString());
```

Comment:
Make the string Static final.

Reply:
How does that matter in this test? This test is deprecated as well.